# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/ClientLayerManager.h
# Commit: 36b0646e8649
# Full Hash: 36b0646e86498662e770756ad54b1fd648a82184
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2020-04-07 21:44:02
# Regressor Bug: 1620594
# File Overlap Count: 1
# Description:
#   Bug 1620594 - Part 7: Remove TabGroup and SystemGroup. r=nika,bas
#   
#   TabGroup never really made any difference in which thread something go
#   dispatched to. This was the intended use, but development of TabGroups
#   with abstract main threads never made it that far. The good thing is
# ==============================================================================

diff -r a5ad3b179c92 -r 36b0646e8649 gfx/layers/client/ClientLayerManager.h
--- a/gfx/layers/client/ClientLayerManager.h	Tue Apr 07 15:17:14 2020 +0000
+++ b/gfx/layers/client/ClientLayerManager.h	Tue Apr 07 15:17:47 2020 +0000
@@ -33,14 +33,8 @@
 class nsDisplayListBuilder;
 
 namespace mozilla {
-
-namespace dom {
-class TabGroup;
-}
 namespace layers {
 
-using dom::TabGroup;
-
 class ClientPaintedLayer;
 class CompositorBridgeChild;
 class ImageLayer;
@@ -66,8 +60,6 @@
 
   ClientLayerManager* AsClientLayerManager() override { return this; }
 
-  TabGroup* GetTabGroup();
-
   int32_t GetMaxTextureSize() const override;
 
   void SetDefaultTargetConfiguration(BufferMode aDoubleBuffering,